src/openvic-simulation/map/Map.cpp

```diff
@@ -562,15 +562,28 @@ bool Map::load_province_positions(BuildingTypeManager const& building_type_manag
 	})(root);
 }
 
-bool Map::load_region_file(ast::NodeCPtr root) {
+bool Map::load_region_colours(ast::NodeCPtr root, std::vector<colour_t>& colours) {
+	return expect_dictionary(
+		[&colours](std::string_view key, ast::NodeCPtr value) -> bool {
+			if (key != "color") {
+				return false;
```

Comment:
```suggestion
				Logger::error("Invalid key in region colours: \"", key, "\"");
				return false;
```
Can you add an error message here